views/lib/footer.hbs

```diff
@@ -9,10 +9,8 @@
 			<div>
 				<a href="/impressum">Impressum</a>
 				<span> - </span>
-				<a href="/datenschutz">Datenschutzerklärung</a>
-				<span> - </span>
 				<a href="{{@root.theme.documents.specificFiles.termsOfUseSchool}}" target="_blank"
-					rel="noopener">Nutzungsordnung</a>
+					rel="noopener">Datenschutzerklärung HPI Schul-Cloud</a>
```

Comment:
Please add this to the localization de.json and en.json.

Reply:
Should I do it for all the footer text?

Reply:
no, it is fine for now if you do it for one key. we have a small workforce doing all the i18n stuff